Makefile

```diff
@@ -43,7 +43,7 @@ all: binary docs
 # Then do the build and the output (skopeo) should appear in current dir
 binary: cmd/skopeo
 	docker build ${DOCKER_BUILD_ARGS} -f Dockerfile.build -t skopeobuildimage .
-	docker run --rm -v ${PWD}:/src/github.com/projectatomic/skopeo \
+	docker run --rm -v $$(pwd):/src/github.com/projectatomic/skopeo:Z \
```

Comment:
FWIW this `:Z` subsumes #150.  (And I have no objections to that, to be explicit.)